pkg/constant/errors.go

```diff
@@ -8,72 +8,72 @@ import (
+	ErrAccountIneligibility                = errors.New("0019")
+	ErrAliasUnavailability                 = errors.New("0020")
+	ErrParentTransactionIDNotFound         = errors.New("0021")
+	ErrImmutableField                      = errors.New("0022")
+	ErrTransactionTimingRestriction        = errors.New("0023")
+	ErrAccountStatusTransactionRestriction = errors.New("0024")
+	ErrInsufficientAccountBalance          = errors.New("0025")
+	ErrTransactionMethodRestriction        = errors.New("0026")
+	ErrDuplicateTransactionTemplateCode    = errors.New("0027")
+	ErrDuplicateAssetPair                  = errors.New("0028")
+	ErrInvalidParentAccountID              = errors.New("0029")
+	ErrMismatchedAssetCode                 = errors.New("0030")
+	ErrChartTypeNotFound                   = errors.New("0031")
+	ErrInvalidCountryCode                  = errors.New("0032")
+	ErrInvalidCodeFormat                   = errors.New("0033")
+	ErrAssetCodeNotFound                   = errors.New("0034")
+	ErrPortfolioIDNotFound                 = errors.New("0035")
+	ErrSegmentIDNotFound                   = errors.New("0036")
+	ErrLedgerIDNotFound                    = errors.New("0037")
+	ErrOrganizationIDNotFound              = errors.New("0038")
+	ErrParentOrganizationIDNotFound        = errors.New("0039")
+	ErrInvalidType                         = errors.New("0040")
+	ErrTokenMissing                        = errors.New("0041")
+	ErrInvalidToken                        = errors.New("0042")
+	ErrInsufficientPrivileges              = errors.New("0043")
+	ErrPermissionEnforcement               = errors.New("0044")
+	ErrJWKFetch                            = errors.New("0045")
+	ErrInternalServer                      = errors.New("0046")
+	ErrBadRequest                          = errors.New("0047")
+	ErrInvalidDSLFileFormat                = errors.New("0048")
+	ErrEmptyDSLFile                        = errors.New("0049")
+	ErrMetadataKeyLengthExceeded           = errors.New("0050")
+	ErrMetadataValueLengthExceeded         = errors.New("0051")
+	ErrAccountIDNotFound                   = errors.New("0052")
+	ErrUnexpectedFieldsInTheRequest        = errors.New("0053")
+	ErrIDsNotFoundForAccounts              = errors.New("0054")
+	ErrAssetIDNotFound                     = errors.New("0055")
+	ErrNoAssetsFound                       = errors.New("0056")
+	ErrNoSegmentsFound                     = errors.New("0057")
+	ErrNoPortfoliosFound                   = errors.New("0058")
+	ErrNoOrganizationsFound                = errors.New("0059")
+	ErrNoLedgersFound                      = errors.New("0060")
+	ErrBalanceUpdateFailed                 = errors.New("0061")
+	ErrNoAccountIDsProvided                = errors.New("0062")
+	ErrFailedToRetrieveAccountsByAliases   = errors.New("0063")
+	ErrNoAccountsFound                     = errors.New("0064")
+	ErrInvalidPathParameter                = errors.New("0065")
+	// 0066 (invalid account type) was removed
```

Comment:
Don't remove this code error.

```suggestion
	ErrInvalidAccountType = errors.New("0066")
```